src/services/CertificateGenerationService.ts

```diff
@@ -507,9 +544,48 @@ class CertificateGenerationService {
                 };
                 console.log("CHECK HERE DOCGENPAYLOAD -> ", docGenPayloadAdr);
                 return docGenPayloadAdr;
+            case CERTIFICATE_DATA.IVA_DATA:
+                const ivaFailDetailsForDocGen = {
+                    SerialNumber: testResult.vehicleType === "trl" ? testResult.trailerId : testResult.vrm,
+                    VehicleTrailerNrNo: testResult.vehicleType === "trl" ? testResult.trailerId : testResult.vrm,
+                    TestCategoryClass: testResult.euVehicleCategory,
+                    TestCategoryBasicNormal: this.isBasicIvaTest(testResult.testTypes.testTypeId) ? IVA_30.BASIC : IVA_30.NORMAL,
+                    Make: testResult.make,
+                    Model: testResult.model,
+                    BodyType: testResult.bodyType,
+                    Date: moment(testResult.testTypes.createdAt).format("DD.MM.YYYY"),
```

Comment:
this should use testTypeStartTimestamp, not created date

Reply:
Updated and repushed.